client/packages/host/src/api/hooks/settings/namePropertyData.ts

```diff
@@ -28,23 +28,104 @@ export type LocalisedNamePropertyConfig = Partial<
   en: ConfigureNamePropertyInput[];
 };
 
-const populationServed: ConfigureNamePropertyInput = {
-  id: '7716cecc-7d62-4f1b-93fa-a55a275079b4',
-  propertyId: POPULATION_SERVED_KEY,
-  key: POPULATION_SERVED_KEY,
-  name: 'Population Served',
-  valueType: PropertyNodeValueType.Float,
-  allowedValues: null,
-  remoteEditable: true,
+const enTranslations: Record<string, string> = {
```

Comment:
Could combine these two maps and make it more generic but this seemed easier for now...